R/metrics-quantile.R

```diff
@@ -310,6 +317,8 @@ interval_coverage <- function(observed, predicted,
 #' The interval coverage deviation is then averaged across all prediction
 #' intervals. The median is ignored when computing coverage deviation.
 #' @inheritParams wis
+#' @importFrom rlang caller_env
```

Comment:
is this used in the function?

Reply:
Nope, my bad. Was left over from initial attempts to include the function call in cli_warn(). I've removed all the `rlang` imports and the package itself from the `Imports` field because it is no longer used.